notebooks/content_understanding_pro_mode.ipynb

```diff
@@ -0,0 +1,248 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load reference storage configuration from environment\n",
+    "REFERENCE_DOC_SAS_URL = os.getenv(\"REFERENCE_DOC_SAS_URL\")\n",
+    "REFERENCE_DOC_PATH = os.getenv(\"REFERENCE_DOC_PATH\")\n",
+    "\n",
+    "await client.generate_knowledge_base_on_blob(reference_docs, REFERENCE_DOC_SAS_URL, REFERENCE_DOC_PATH, skip_analyze=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create analyzer with defined schema for Pro mode\n",
+    "Before creating the custom fields analyzer, you should fill the constant ANALYZER_ID with a business-related name. Here we randomly generate a name for demo purpose.\n",
+    "\n",
+    "We use **REFERENCE_DOC_SAS_URL** and **REFERENCE_DOC_PATH** that's set up in `.env` file and used in the previous step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import uuid\n",
+    "CUSTOM_ANALYZER_ID = \"pro-mode-sample-\" + str(uuid.uuid4())\n",
+    "\n",
+    "response = client.begin_create_analyzer(\n",
+    "    CUSTOM_ANALYZER_ID,\n",
+    "    analyzer_template_path=analyzer_template,\n",
+    "    pro_mode_reference_docs_storage_container_sas_url=REFERENCE_DOC_SAS_URL,\n",
+    "    pro_mode_reference_docs_storage_container_path_prefix=REFERENCE_DOC_PATH,\n",
+    ")\n",
+    "result = client.poll_result(response)\n",
+    "if result is not None and \"status\" in result and result[\"status\"] == \"Succeeded\":\n",
+    "    logging.info(f\"Here is the analyzer detail for {result['result']['analyzerId']}\")\n",
+    "    logging.info(json.dumps(result, indent=2))\n",
+    "else:\n",
+    "    logging.info(\n",
+    "        \"Check your service please, may be some issues in configuration and deployment\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use created analyzer to do data analysis of the input documents\n",
+    "After the analyzer is successfully created, we can use it to analyze our input files.\n",
+    "> NOTE: Pro mode does multi-step reasoning and may take longer time to analyze."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = client.begin_analyze(CUSTOM_ANALYZER_ID, file_location=input_docs)\n",
+    "result_json = client.poll_result(response, timeout_seconds=360)\n",
+    "\n",
+    "logging.info(json.dumps(result_json, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Delete exist analyzer in Content Understanding Service\n",
+    "This snippet is not required, but it's only used to prevent the testing analyzer from residing in your service. The custom fields analyzer could be stored in your service for reusing by subsequent business in real usage scenarios."
```

Comment:
Reword this sentence: Without deletion, the analyzer will remain in your service for subsequent reuse.